### DISCUSSION TITLE
Limit orders - Binance.US

### DISCUSSION THREAD

**Discussion by sirouk** [thibaultyou/tradingview-alerts-processor#48] [Ideas]
Thanks so much for your work!

I would really like to see if we can get a limit order working with this bridge for Binance.US

You have it listed as something to do, just wondering how I can help!
https://github.com/thibaultyou/tradingview-alerts-processor/projects/1#card-63871341

**Comment by leifjones** [comment]
I don't think anyone is actively working on it, but we'd likely review a PR. I'd like to have it as well!

Try to do it in a exchange-agnostic way

**Reply by sirouk** [reply]
Cool beans. I'll give it a try. Would you be able to point me in the right direction, perhaps with some technical documentation? Or should I just open the scripts and piece together the context of what fires for a trade?

**Reply by leifjones** [reply]
Sorry to say what you see is all there is. When I was working with it a year or two ago, I think I used Postman to poke at it as I was teaching myself what was going on. I think I used VS Code’s Node debugging
